packages/app/src/components/Root/Root.tsx

```diff
@@ -248,49 +272,57 @@ export const Root = ({ children }: PropsWithChildren<{}>) => {
     );
   };
   return (
-    <SidebarPage>
-      <Sidebar>
-        <SidebarLogo />
-        <SidebarGroup label="Search" icon={<SearchIcon />} to="/search">
-          <SidebarSearchModal />
-        </SidebarGroup>
-        <SidebarDivider />
-        <SidebarGroup label="Menu" icon={<MuiMenuIcon />}>
-          {/* Global nav, not org-specific */}
-          {renderMenuItems(true, false)}
-          {/* End global nav */}
+    <>
+      {headerPositions.includes('above-sidebar') && (
+        <ApplicationHeader headerMountPoints={headerMountPoints} />
+      )}
```

Comment:
We should also allow both headers in parallel:

```suggestion
      <ApplicationHeader mountPoints={headers} position="above-sidebar" />
```